app/src/main/res/values/dimens.xml

```diff
@@ -588,6 +588,11 @@
   <dimen name="section_title_text_view_layout_margin_top">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">32dp</dimen>
 
+  <!-- Topic Practice Fragment -->
```

Comment:
Nit
`<!-- TopicPracticeFragment -->`

here and everywhere else in other files.

Reply:
Done.